### DISCUSSION TITLE
Paypal Express Checkout - Shipping Address Issue

### DISCUSSION THREAD

**Discussion by Spike00** [zencart/zencart#5809] [Ideas]
The issue and a workaround (not yet tested, I'll test next days since I have a site duffering from ths issue) are explained here: https://www.zen-cart.com/showthread.php?186414-Paypal-Express-Checkout-Shipping-Address-Issue/page5&highlight=ADDROVERRIDE

**Comment by drbyte** [comment]
While researching this I came up with several questions, which I've posted here: https://www.zen-cart.com/showthread.php?186414-Paypal-Express-Checkout-Shipping-Address-Issue&p=1288435#post1288435

**Comment by drbyte** [comment]
Requires more research.
